frontend/app/components/date-of-birth.tsx

```diff
@@ -0,0 +1,42 @@
+import { InputField } from './input-field';
+import { InputSelect } from './input-select';
+
+export interface DateOfBirthProps {
+  lang: string;
+  legend: string;
+  monthLabel: string;
+  dayLabel: string;
+  yearLabel: string;
+  monthDefault?: number;
+  dayDefault?: number;
+  yearDefault?: number;
+  errorMessageMonth?: string;
+  errorMessageDay?: string;
+  errorMessageYear?: string;
+}
+
+export function DateOfBirth(props: DateOfBirthProps) {
+  const { lang, legend, monthLabel, dayLabel, yearLabel, monthDefault, dayDefault, yearDefault, errorMessageMonth, errorMessageDay, errorMessageYear } = props;
+  const monthOptions = Array.from({ length: 12 }, (_, i) => ({ children: new Intl.DateTimeFormat(`${lang}-ca`, { month: 'long' }).format(new Date(1900, i, 1)), value: i, id: `month-${i}` }));
+
+  return (
+    <fieldset>
+      <legend className="mb-2 font-semibold">{legend}</legend>
+      <div className="flex flex-col gap-6 sm:flex-row">
+        {lang == 'fr' && (
```

Comment:
```suggestion
        {lang === 'fr' && (
```

(same comment for `'en'` below)